core/autoscaling_configuration.go

```diff
@@ -243,7 +243,7 @@ func (a *autoScalingGroup) loadConfSpot() bool {
 	}
 	if newValue, done := a.loadBiddingPolicy(tagValue); done {
 		a.region.conf.BiddingPolicy = newValue
-		logger.Println("BiddingPolicy =", a.region.conf.BiddingPolicy)
+		debug.Println("BiddingPolicy =", a.region.conf.BiddingPolicy)
```

Comment:
I don't think this adds anything since the default bidding policy was already logged on startup.